### PR TITLE
Adding flavor specific changes to finetune script

### PR DESCRIPTION
**Pull request by Anubha98**
Adding flavor specific changes to finetune script

image

### CODE REVIEW

**Review by skanakamedal — APPROVED** [review]

**Review by skanakamedal — CHANGES_REQUESTED** [review]
Fix the PR syntax gate, will approve then

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py [hunk 1]
```diff
@@ -807,18 +843,31 @@ def finetune(args: Namespace):
...
-        mlflow_ft_conf["mlflow_model_signature"] = deep_update(
-            mlflow_ft_conf["mlflow_model_signature"],
-            MLFLOW_MODEL_SIGNATURES[args.task_name],
-        )
-        logger.info(
-                    f"Adding mlflow model signature for task {args.task_name} - "
-                    f"{MLFLOW_MODEL_SIGNATURES[args.task_name]}"
-                )
+    mlflow_data = _load_mlflow_model(args.model_selector_output)
+    current_flavor = None
```
Comment (Anubha98) [inline assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:850]: rename: mlflow_flavor

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py [hunk 2]
```diff
@@ -927,6 +965,46 @@ def finetune(args: Namespace):
     hf_task_runner = get_task_runner(task_name=args.task_name)()
     hf_task_runner.run_finetune(args)
 
+    # remove unwanted packages and add pinned packages in requirements.txt and conda.yaml
```
Comment (Anubha98) [inline assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:968]: ToDo, workaround for now, we need to find better solution

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py (hunk 1 above)
Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py]: fixed

**Review by jpmann — APPROVED** [review]

**Review by skanakamedal — APPROVED** [review]

**Pull request merged**